snuba/cli/consumer.py

```diff
@@ -19,8 +19,10 @@
               help='Consumer group use for consuming the raw events topic.')
 @click.option('--bootstrap-server', default=settings.DEFAULT_BROKERS, multiple=True,
               help='Kafka bootstrap server to use.')
-@click.option('--clickhouse-server', default=settings.CLICKHOUSE_SERVER,
+@click.option('--clickhouse-host', default=settings.CLICKHOUSE_HOST,
               help='Clickhouse server to write to.')
+@click.option('--clickhouse-native-port', default=settings.CLICKHOUSE_NATIVE_PORT, type=int,
```

Comment:
On the fence about whether this should be `--clickhouse-port` — this disambiguates it from `--clickhouse-http-port` to come later, but maybe `-native` is a bit redundant?